README.md

```diff
@@ -144,6 +144,28 @@ job :kitto_last_commit,
     command: "curl https://api.github.com/repos/kittoframework/kitto/commits\?page\=1\&per_page\=1"
 ```
 
+## Hooks
+
+If, instead of polling for new data from a data source, you want to act on data
+as it changes, hooks are a useful feature for implementing webhooks to feed data
```

Comment:
> hooks are a useful feature for implementing webhooks

I find this a bit difficult to comprehend. Considering that someone may not be familiar with the concept of hooks, having `hooks` and `webhooks` in the same sentence can be a bit confusing.